src/models/environment.ts

```diff
@@ -147,6 +148,15 @@ export class CCloudEnvironment extends Environment {
       pool instanceof CCloudFlinkComputePool ? pool : new CCloudFlinkComputePool(pool),
     );
   }
+  /**
+   * Flinkable Kafka clusters, i.e. those with at least one associated Flink compute pool.
+   */
+  get flinkDatabaseClusters(): CCloudFlinkDbKafkaCluster[] {
+    return this.kafkaClusters.filter(
+      (kc): kc is CCloudFlinkDbKafkaCluster =>
+        kc.isFlinkable() && Array.isArray(kc.flinkPools) && kc.flinkPools.length > 0,
```

Comment:
Thanks! Convenience!

I don't think you need the ` && Array.isArray... ` bit; the `kc.isFlinkable()` portion of the filter is all that is necessary (`CCloudKafkaCluster` vs its Flink pools data member is exactly what `kc.isFlinkable()` is doing).